src/index.html

```diff
@@ -13,6 +13,49 @@
     />
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars">
+      <div class="stars stars--0">
+        <p class="stars__star"></p>
+        <p class="stars__star"></p>
+        <p class="stars__star"></p>
+        <p class="stars__star"></p>
+        <p class="stars__star"></p>
+      </div>
+      <div class="stars stars--1">
```

Comment:
Add the blank line between the blocks
```suggestion
      </div>
      
      <div class="stars stars--1">
```